kits/java/src/main/java/com/luxai/lux/action/UnitAction.java

```diff
@@ -0,0 +1,22 @@
+package com.luxai.lux.action;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class UnitAction {

```

Comment:
Super small nitpick. `UnitActions` would be more appropriate.

Reply:
fix